app/actors/SessionsScheduler.scala

```diff
@@ -254,16 +254,18 @@ class SessionsScheduler @Inject()(sessionsRepository: SessionsRepository,
         }
       }
     }
-
     reminderEmailHandlerForPresenter(sessions, emailInfo, presenterEmails)
   }
 
   def reminderEmailHandlerForPresenter(sessions: List[SessionInfo], emailInfo: List[EmailInfo], presenterEmails: List[String]): Unit = {
     presenterEmails foreach { presenterEmail =>
       val presenterOtherTopic = emailInfo.filterNot(_.presenter == presenterEmail)
-      presenterOtherTopic foreach { topic =>
+      if(presenterOtherTopic.nonEmpty){
```

Comment:
Not sure if this point was clear when communicated earlier, the changes needs to be properly formatted before sending pull requests.

Reply:
Okay Sir I have removed all the unnecessary extra lines.